dandi/dandiapi.py

```diff
@@ -437,7 +447,13 @@ def iter_upload(self, dandiset_id, version_id, asset_metadata, filepath):
                             part["part_number"],
                             part["size"],
                         )
-                        r = storage.put(part["upload_url"], data=chunk, json_resp=False)
+                        chunk_md5 = md5(chunk).digest()
```

Comment:
note to myself: so we need it per chunk, thus cannot (ab)use Digester to get both md5 and sha256 at the same time.  Here at least chunk will already be in memory so should not be much of a burden.